### PR TITLE
Home Page: Modified the statements in more meaningful way

### PR DESCRIPTION
**Pull request by nutboltu**
- Modified the statements in more meaningful way
- Changed few words in proper Bangla

### CODE REVIEW

File: content/home/examples/a-simple-component.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ order: 0
 domid: hello-example
 ---
 
-React কম্পোনেন্টগুলি একটি `render()` মেথড ব্যবহার করে যা ইনপুট ডেটা নেয় এবং কী দেখাতে হবে তা রিটার্ন করে। এই উদাহরণটি JSX নামে একটি XML এর মত একটি সিনট্যাক্স ব্যবহার করে। কম্পোনেন্টটে যেই ইনপুট ডেটা দেওয়া হয়, সেগুলো `this.props` এর মাধ্যমরে `render()` মেথডের ভেতরে ব্যবহার করে যায়।
+React কম্পোনেন্ট একটি `render()` মেথড তৈরি করে যা ইনপুট ডেটা নেয় এবং কী দেখাতে হবে তা প্রদান করে। এই উদাহরণটি JSX নামে একটি XML এর মত সিনট্যাক্স ব্যবহার করেছে। কম্পোনেন্টটে যেই ইনপুট ডেটাই দেওয়া হয়, তা `this.props` এর মাধ্যমে `render()` মেথডের ভেতরে ব্যবহার করা যায়।
 
-**JSX একটি অপশনাল সিনট্যাক্স এবং React এ ব্যবহার করা আবশ্যিক নয়।** JSX এর কম্পাইলেশন ধাপে, যেই রো জাভাস্ক্রিপ্ট তৈরি হয়, সেগুলো পর্যবেক্ষণ করতে [Babel REPL](babel://es5-syntax-example) ব্যবহার করতে পারেন।
\ No newline at end of file
+**JSX একটি অপশনাল সিনট্যাক্স এবং এটি React এ ব্যবহার করা আবশ্যিক নয়।** JSX এর কম্পাইলেশন ধাপে, যেই কাঁচা জাভাস্ক্রিপ্ট তৈরি হয়,  [Babel REPL](babel://es5-syntax-example) ব্যবহার করে আপনি তা পর্যবেক্ষণ করতে পারেন।
```
Comment (zpbappi) [inline content/home/examples/a-simple-component.md]: I would remove কাঁচা from the translation. Only "Javascript" is conveying the meaning better.
যেই <del>কাঁচা</del> জাভাস্ক্রিপ্ট

File: content/home/marketing/learn-once-write-anywhere.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-title: একবার শিখুন, যে কোনোখানে লিখুন
+title: একবার শিখুন, যে কোন স্থানে লিখুন
 order: 2
 ---
 
-আমরা আপনার বাকি প্রযুক্তি স্ট্যাক সম্পর্কে অনুমান করি না, এজন্যে আপনি বর্তমান কোডে কোন পরিবর্তন ছাড়াই নতুন ফিচার React এ ডেভেলপ করতে পারেন।
+আমরা আপনার বাকি প্রযুক্তি স্ট্যাকের উপর নির্ভর করি না, এজন্য আপনি বর্তমান কোডে কোন পরিবর্তন ছাড়াই React দিয়ে নতুন ফিচার তৈরি করতে পারেন।
 
-Node এবং পাওয়ার মোবাইল অ্যাপ্লিকেশন [React Native](https://facebook.github.io/react-native/) ব্যবহার করে সার্ভারে React রেন্ডার করতে পারেন।
+Node এবং [React Native](https://facebook.github.io/react-native/) দিয়ে বানানো শক্তিশালী মোবাইল অ্যাপ্স ব্যবহার করে আপনি সার্ভারেও React রেন্ডার করতে পারেন।
```
[the remark this answers is not in the document]
Comment (nutboltu) [reply content/home/marketing/learn-once-write-anywhere.md]: I changed it to 
```
Node ব্যবহার করে আপনি সার্ভারেও React রেন্ডার করতে পারেন এবং এটি [React Native](https://facebook.github.io/react-native/) দিয়ে মোবাইল অ্যাপ্স তৈরিতেও সাহায্য করে।
```

**Comment by nutboltu** [comment]
@zpbappi I addressed you feedback. Please have another look

**Comment by zpbappi** [comment]
@nutboltu Great Job!

**Pull request merged**